notification/build.gradle

```diff
@@ -19,7 +19,7 @@ apply plugin: 'jacoco'
 dependencies {
     compileOnly "org.elasticsearch:elasticsearch:${es_version}"
     compile "org.apache.httpcomponents:httpcore:4.4.5"
-    compile "org.apache.httpcomponents:httpclient:4.5.7"
+    compile "org.apache.httpcomponents:httpclient:4.5.8"
```

Comment:
it this upgrade necessary and required?

Reply:
discussed 
[July 24, 2019, 11:34 AM] Qureshi, Mohammad: Yeah, the build failed because there was a version conflict, the new ES version depends on httpclient 4.5.8

Reply:
And just to briefly expound on that, I tried to change this to use [ES's versions](https://github.com/elastic/elasticsearch/blob/39d0355bca37d38349058da51948c6cc46d2a7f2/buildSrc/version.properties) (ex. ```${versions.httpclient}```) instead of harcoding the version but that requires adding ```apply plugin: 'elasticsearch.esplugin'``` which is primarily for ES plugins not common Java libraries, so leaving the hardcoded versions for now.